src/Oscar.jl

```diff
@@ -117,9 +117,31 @@ function example(s::String)
   Base.include(Main, joinpath(oscardir, "examples", s))
 end
 
-function build_doc()
+function doc_init()
+  old = Pkg.project().path
+  Pkg.activate(joinpath(oscardir, "docs"))
   Base.include(Main, joinpath(oscardir, "docs", "make_local.jl"))
+  Pkg.activate(old)
+end
+
+function doc_update_deps()
+  old = Pkg.project().path
+  Pkg.activate(joinpath(oscardir, "docs"))
+  Pkg.update()
+  Pkg.activate(old)
```

Comment:
Would this also work?
```suggestion
  Pkg.activate(joinpath(oscardir, "docs")) do
    Pkg.update()
  end
```